go.mod

```diff
@@ -22,13 +22,13 @@ require (
 	k8s.io/api v0.22.5
 	k8s.io/apimachinery v0.22.5
 	k8s.io/client-go v0.22.5
-	knative.dev/eventing v0.29.0
-	knative.dev/eventing-kafka v0.27.0
+	knative.dev/eventing v0.30.1
+	knative.dev/eventing-kafka v0.28.0
```

Comment:
why 0.30 VS 0.28 ?

Reply:
I don't set it manually, `hack/update-deps.sh --upgrade` does the update.